package.json

```diff
@@ -23,49 +66,50 @@
   "devDependencies": {
     "Base64": "0.3.0",
     "aws-sdk": "2.2.44",
-    "babel-eslint": "5.0.0",
+    "babel-cli": "6.11.4",
+    "babel-core": "6.13.2",
+    "babel-eslint": "6.1.2",
+    "babel-plugin-add-shopify-header": "1.0.3",
+    "babel-plugin-eslint-test-generator": "1.0.1",
+    "babel-plugin-rename-umd-globals": "0.0.3",
     "babel-plugin-transform-es2015-modules-amd": "6.8.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "6.8.0",
-    "babel-plugin-transform-object-assign": "6.8.0",
+    "babel-plugin-transform-es2015-modules-umd": "6.12.0",
+    "babel-plugin-transform-inline-environment-variables": "6.8.0",
     "babel-preset-es2015": "6.9.0",
-    "babel-register": "6.9.0",
-    "broccoli": "0.16.9",
-    "broccoli-asset-rev": "2.4.1",
-    "broccoli-babel-transpiler": "git+ssh://git@github.com/babel/broccoli-babel-transpiler.git#17b304da9a1c2c93ff484e3d37029f5fad0f96c9",
-    "broccoli-cli": "1.0.0",
-    "broccoli-concat": "2.2.0",
-    "broccoli-env": "0.0.1",
-    "broccoli-funnel": "1.0.1",
-    "broccoli-lint-eslint": "2.4.1",
-    "broccoli-merge-trees": "1.1.1",
-    "broccoli-plugin": "1.2.1",
-    "broccoli-sane-watcher": "1.1.4",
-    "broccoli-sass-source-maps": "1.7.0",
-    "broccoli-static-compiler": "0.2.2",
-    "broccoli-uglify-js": "0.1.3",
-    "broccoli-uglify-sourcemap": "1.1.1",
-    "escope": "3.3.0",
+    "concurrently": "2.2.0",
+    "core-js": "~2.4.1",
```

Comment:
Please pin this version. I'd rather not have drifting sets of functionality outside of our control.